### ISSUE TITLE
Default template is missing newline after highlight note, can lead to merging of notes

### ISSUE THREAD

**Issue by gwillcox-r7** [kaiiiz/obsidian-raindrop-highlights-plugin#17]
The default template has the following line of code which can cause the highlights notes to merge together:

https://github.com/kaiiiz/obsidian-raindrop-highlights-plugin/blob/d76d717b943a1fc8cd8c97b5386dc6aecebeeb09/src/assets/defaultTemplate.njk#L22-L27

The problem with this code however is that it fails to append a new line after the highlight note itself. This leads to issues as Obsidian will interpret the `> [!{{callout}}]+`  part of the line as being part of the previous highlight's note as well, leading to them all joining into one giant note as can be seen below:

![image](https://user-images.githubusercontent.com/63261883/191652923-94acb113-e23e-41ad-a640-fe59c6143683.png)

Lets see what the raw file contains since this is preview mode:

![image](https://user-images.githubusercontent.com/63261883/191653037-11236581-7da1-4af6-bd0c-57c91ea3aaf8.png)

Looks like there are missing new lines between some of these elements.

To fix this we can use the following code, which just adds a new line after the end of the highlight note itself:

```
> [!{{callout}}]+ Updated on {{highlight.lastUpdate}}
>
> {{highlight.text.split("\n") | join("\n>")}}
{% if highlight.note -%}> > {{highlight.note + "\n"}}{%- endif %}

{%- endfor -%}
```

And here are the results on the same set of notes:

![image](https://user-images.githubusercontent.com/63261883/191653307-daeeef2a-12a5-4f99-9944-f77708580332.png)

Happiness is achieved once again 😄

**Comment by kaiiiz** [comment]
Thanks for reporting this bug. I'll fix it in the next release.

**Comment by kaiiiz** [comment]
Fix in 0.0.15
